app/i18n/fr/gen.php

```diff
@@ -229,6 +229,7 @@
 		'pocket' => 'Pocket',	// IGNORE
 		'print' => 'Imprimer',
 		'raindrop' => 'Raindrop.io',	// IGNORE
+		'readeck' => 'Readeck',
```

Comment:
```suggestion
		'readeck' => 'Readeck',	// IGNORE
```

And likewise for the other languages, except `en`